submissions/benjaminKamena/Dom/product.js

```diff
@@ -0,0 +1,40 @@
+
```

Comment:
@kamena1994 Remove this file from this pull request the task to work on this will come in later.